core/controllers/topic_editor.py

```diff
@@ -199,6 +203,8 @@ def get(self, topic_id: str) -> None:
                 'total_chapters_count': total_chapters_count,
                 'published_chapters_count': published_chapters_count,
                 'upcoming_chapters_count': upcoming_chapters_count,
+                'upcoming_chapters_expected_days': (
```

Comment:
You should probably sort this list after populating it, right?

Reply:
Done